components/config/prefs.rs

```diff
@@ -297,11 +297,15 @@ mod gen {
                     enabled: bool,
                     #[serde(default)]
                     test: bool,
-                    #[serde(default)]
-                    glwindow: bool,
+                    glwindow: {
+                        #[serde(default)]
+                        enabled: bool,
+                    },
                     hands: {
                         #[serde(default)]
                         enabled: bool,
+                        #[serde(rename = "dom.webxr.glwindow.red-cyan")]
+                        red_cyan: bool,
```

Comment:
Ooh, red-cyan hands!

Reply:
Thank you rebase conflict resolution. Fixed now.